src/SymfonyRequestMessage.php

```diff
@@ -2,22 +2,37 @@
 
 namespace HttpSignatures;
 
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
Cool, I also ran it through `php-cs-fixer` (I'll push the result to this branch) and the `use` statements didn't vanish, so I guess that's considered normal.